send_mail.py

```diff
@@ -0,0 +1,110 @@
+""" Basic usage of smtplib For
+Sending mails to LinkinPark Forum.
+Get the receiver's mail from file(message)
+
+Method 1:
+---------
+    1. smtplib module (smtp or esmtp: Extended smtp)
+        -> Establish a connection
+        -> Create a MIME text/image obj. from scratch
+        -> Send mail()
+        -> Quit()
+
+"""
+
+import ssl # for creating context - CA certificates
+import smtplib
+import logging
+from email import encoders # for encoding the data into base64
+from email.mime.base import MIMEBase # Base = application/octet-stream
+from email.mime.multipart import MIMEMultipart  # multipart = mixed
+from email.mime.text import MIMEText  # multipart = text/plain
+from email.policy import default  # Make a use of RFC and \n For line ending
+
+
+class SendToLPForum:
+    """ Send mails to LinkinPark Forum """
+
+    def __init__(self, username, password):
+        self.user = username
+        self.passwd = password
+        self.host = "smtp.gmail.com"
+        self.port = 587  # Port (ssl=465, tls=587)
+
+    def login(self):
+        """ Login to the smtp server """
+
+        # load the system trusted CA certificates
+        # enable host checking and certificates validation
+        try:
+            context = ssl.create_default_context()
+
+            server = smtplib.SMTP(self.host, self.port)
+            server.starttls(context=context) # secured by tls connection
+        
+            server.login(self.user, self.passwd)
+            return server
+        except Exception as err:
+            logging.info(f"SendToLPForum.login : {err}")
```

Comment:
logging.error ?

Seems like it won't work, so its more serious than info

Reply:
great, i'll replace all the instances of logging.info with logging.error and save it inside a file 
Thankss